src/test/groovy/graphql/execution/preparsed/persisted/InMemoryPersistedQueryCacheTest.groovy

```diff
@@ -50,8 +50,8 @@ class InMemoryPersistedQueryCacheTest extends Specification {
                 .build()
         def ei = mkEI(hash, PersistedQuerySupport.PERSISTED_QUERY_MARKER)
         when:
-        def getDoc = inMemCache.getPersistedQueryDocument(hash, ei, onMiss)
-        def doc = getDoc.document
+        def getDoc = inMemCache.getPersistedQueryDocumentAsync(hash, ei, onMiss)
+        def doc = getDoc.get().document
```

Comment:
Can we have one test that goes via the old deprecated path so that we have surety that while its alive it works